.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .temp
 vuepress
 TODOs.md
+*.sw*
```

Comment:
Why adding this?

Reply:
For vim swap files.

Reply:
You should add it at your project instead of here.

Reply:
Just thought it would be great since there might be other contributors who use vim.

Reply:
Hmmm, so should I put all the different needs here?

Reply:
It doesn't depend on me. please do not kidnap thinking.

Just I believe most VIM programmers have set this to a global configuration instead of duplicate it at every project.

Reply:
That is certainly not true for me. I am happy to remove that if this is not wanted.